src/components/helpers.js

```diff
@@ -83,6 +84,7 @@ import UnknownIcon from 'cozy-ui/transpiled/react/Icons/Unknow'
 import WalkIcon from 'cozy-ui/transpiled/react/Icons/Walk'
 import WorkIcon from 'cozy-ui/transpiled/react/Icons/Work'
 
+const logService = logger.namespace('services/recurringPurposes')
```

Comment:
That does not look like a recurringPurpose service ;)

Reply:
En effet, c'est le service `timeseriesWithoutAggregateMigration` qui utilise ce log, j'ai revu aussi l'implem